mkdocs.yml

```diff
@@ -19,6 +19,7 @@ nav:
     - creating-charts.md
     - predbat-plan-card.md
   - 'Predbat development':
+    - other-inverters.md
```

Comment:
Personally I'd add this in another section, rather than in the section on developing on Predbat, as it's not really developer content.